Recommend users to rename their fork's default branch (#1734)
rust-lang/blog.rust-lang.org: content/inside-rust/renaming-the-default-branch-of-rust-lang-rust.md

If you have a fork of the `rust-lang/rust` repository on GitHub and would like to rename your default branch to match, you can follow [GitHub's instructions][github-how-to-rename].

> **We recommend you to rename the default branch of your fork.** If you do not do it, and you will some day type `git checkout master` from muscle memory, git will create a fresh `master` branch based on your fork's `master` branch, which will be outdated. This behavior can be quite confusing.

Reviewer: ```suggestion
**We recommend renaming the default branch of your fork.** If you do not rename it and later run `git checkout master`, git will create a`master` branch based on your fork's outdated `master` branch. This behavior can be confusing.
```
